crates/revm/src/inspector.rs

```diff
@@ -8,13 +8,15 @@ use auto_impl::auto_impl;
 pub mod customprinter;
 pub mod gas;
 pub mod noop;
+pub mod tracer_eip3155;
```

Comment:
```suggestion
#[cfg(feature = "std")]
pub mod tracer_eip3155;
```
It is better to be behind the std flag.
as it uses `std::io::Write` that needs std bcs of io::Error: https://github.com/rust-lang/rust/issues/68315

Reply:
done